cmd/healthbot/config.go

```diff
@@ -22,17 +22,18 @@ type config struct {
 		Human bool `default:"false"`
 		Debug bool `default:"false"`
 	}
-	Target string `default:""`
 	Chains []ChainConfig
 }
 
 // ChainConfig contains probe configuration for a particular chain.
 type ChainConfig struct {
-	Name  string
+	Name   string
+	Signer struct {
+		PrivateKey string `default:""`
+	}
 	Probe struct {
 		CheckInterval  string `default:"15s"`
 		ReceiptTimeout string `default:"20s"`
-		SIWE           string `default:""`
 		Tablename      string `default:""`
 	}
 }
```

Comment:
I'm making some assumptions here about the possible structure of the json file that is read for the healthbot config. `docker/local/api/config.json` includes a `Signer.PrivateKey` property that I'm leveraging. I see that `docker/deployed/testnet/healthbot/config.json` doesn't have that property, but I'm hoping it would be ok to add it. There is no need for `Probe.SIWE` because the Go client creates the SIWE string internally by using the provided private key. Let's talk about this.